packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-state.ts

```diff
@@ -335,3 +350,17 @@ const getPolygonCoords = ({
 
     return coords;
 };
+
+/**
+ * determine radius of a circle graph
+ *
+ * @param graph - the graph object containing the circle
+ * @returns the radius of the circle
+ */
+export function getRadius(graph: CircleGraphState): number {
+    const [centerX, centerY] = graph.center;
+    const [edgeX, edgeY] = graph.radiusPoint;
+    return Math.sqrt(
+        Math.pow(edgeX - centerX, 2) + Math.pow(edgeY - centerY, 2),
+    );
```

Comment:
The Pythagorean theorem feels like overkill here, but I guess it's less likely to break in the future.

Reply:
I personally like it for being future-proof.